timedomain/sp_utils.py

```diff
@@ -137,22 +176,31 @@ def postDESIInstrument():
     def test():
 
         
-#         response = SkyPortal.api('DELETE', '{}/api/candidates/{}'.format(SkyPortal.url,"35191288252861933"))
+        response = SkyPortal.api('DELETE', '{}/api/candidates/{}'.format(SkyPortal.url,"35191288252861933"))
 
-#         print(f'HTTP code: {response.status_code}, {response.reason}')
-#         if response.status_code in (200, 400):
-#             print(f'JSON response: {response.json()}')
+        print(f'HTTP code: {response.status_code}, {response.reason}')
+        if response.status_code in (200, 400):
+            print(f'JSON response: {response.json()}')
             
         filename = fitsfile("70006","20200305", "6", subdir='andes',trunk='coadd') 
-#         fibermap = Table.read(filename, 'FIBERMAP')
-#         index = np.where(fibermap['TARGETID'].data== 35191288252861933)[0]
-#         SkyPortal.postCandidate(index[0],fibermap)
+        fibermap = Table.read(filename, 'FIBERMAP')
+        index = np.where(fibermap['TARGETID'].data== 35191288252861933)[0]
+        SkyPortal.postCandidate(index[0],fibermap)
         
         spectra = read_spectra(filename)
-#         SkyPortal.postSpectra("35191288252861933",spectra)
+        SkyPortal.postSpectra("35191288252861933",spectra)
 
         
         
-# SkyPortal.instrument_id()
+SkyPortal.instrument_id()
 
-# SkyPortal.test()
\ No newline at end of file
+SkyPortal.test()
+
+    @staticmethod
+    def test2():
+        print(SkyPortal.filter_id("DESI Difference CV"))
+        print(SkyPortal.filter_id("AMPEL.HU_RANDOM"))
+        print(SkyPortal.filter_id("DESIDIFF_CV_daily"))
+        print(SkyPortal.filter_id("xyz"))
+        
+SkyPortal.test2()
```

Comment:
This will run, which we do not want to be default behavior.  So this should be commented out.